pkg/steps/release/create_release.go

```diff
@@ -175,6 +175,11 @@ func (s *assembleReleaseStep) run(ctx context.Context) error {
 		return results.ForReason("missing_release").WithError(err).Errorf("could not resolve imagestream %s: %v", streamName, err)
 	}
 
+	// we want to expose the release payload as a CI version that looks just like
+	// the release versions for nightlies and CI release candidates
+	now := time.Now().UTC().Truncate(time.Second)
+	version := fmt.Sprintf("%s.0-0.test-%s", s.config.Name, now.Format("2006-01-02-150405"))
```

Comment:
Commented in slack, but it might be good to add the release inputs hash (9 hex chars?) at the end `4.5.0-0.test-DATE-HASH`